app/models/api_key.rb

```diff
@@ -1,5 +1,6 @@
+# ApiKey
 class ApiKey < ActiveRecord::Base
-  belongs_to  :user
+  belongs_to :user
 
-  validates_presence_of   :token, :user_id
+  validates_presence_of :token, :user_id
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.